www/plugins.njk

```diff
@@ -0,0 +1,169 @@
+        </g>
+      </g>
+    </svg>
+    <h1 class="header-snowpack">Snowpack</h1>
+    <p class="header-snowpack-subtitle">
+      The Snowpack Plugin Catalog
+    </p>
+  </div>
+  <div class="content">
+    <div style="text-align: center; margin: 1rem 0;">
+      <p>Browse plugins that will make your blazing fast site even more awesome.</p>
+      <p>To learn more about Snowpack plugins, read our full plugin <a href="/#plugins">getting started</a> guide.</p>
+      <p>To create your own plugin, check out the <a href="/guides/plugins">Plugin Authoring</a> guide.</p>
+      <p id="total-result-count"></p>
+
+    </div>
+    <form name="myform" id="myform" class="sky-form" action="https://www.npmjs.com/search" method="GET">
+      <input type="search" name="q" value="" placeholder="search Sass, sitemaps, image optimization..." class="svelte-36nwx5" style="
+margin: 0;
+padding: 0.25rem;
+border-radius: 2px 0 0 2px;
+border: 1px solid #BBB;
+display: block;
+font-weight: 500;
+font-size: 22px;
+transition: color 150ms linear;
+max-width: 60ch;
+flex: 1;
+">
+      <button type="submit" class="sky-btn">Search</button>
+    </form>
+
+    <br/>
+    <h2 style="text-align: center; font-size: 1.75rem; font-weight: bold;">Popular Snowpack Plugins</h2>
+
+    <section id="search-results">
+      <div id="loading-message">Loading...</div>
+    </section>
+  </div>
+
+  <script type="module">
+    import {html, render} from 'https://cdn.skypack.dev/htm/preact/standalone.module.js'
+
+    function Card({result}) {
+      return html ` <li class="card">
+      <img class="plugin-icon" src="/img/rocket-solid.svg" />
+      <header class="card-header">
+        <h3 class="card-name">
+          <a href="https://www.skypack.dev/view/${result
+        .name}" target="_blank"><span itemprop="name">${result
+        .name}</span></a>
+        </h3> 
+      </header> 
+      <p class="card-descr" itemprop="description">${result
+        .description
+        .split('. ')[0]}</p> 
+      <p class="card-subtitle">
+		    Updated <time class="" datetime="${result
+        .updatedAt}">August 10, 2020</time>
+      </p>
+    </li>`;
+    }
+
+    document
+      .getElementById('myform')
+      .onsubmit = function DoSubmit(e) {
+        const form = new FormData(e.target);
+        const formula = form.get("q");
+        document
+          .getElementById('loading-message')
+          .style.display = 'block';
+        searchPlugins(formula);
+        return false;
+      }
+
+    async function searchPlugins(val) {
+      const res = await fetch(`https://api.skypack.dev/v1/search?q=${encodeURIComponent('snowpack plugin ' + (
```

Comment:
Maybe specify a higher `count` as currently only 25 will be returned (you can use `new URLSearchParams()` to easily set/join these)